src/index.ts

```diff
@@ -24,6 +24,8 @@ export function parseLatex(value: string) {
         ddot: { signature: 'm' },
         hat: { signature: 'm' },
         widehat: { signature: 'm' },
+        overset: { signature: 'm m' },
+        underset: { signature: 'm m' },
```

Comment:
@fwkoch changed to this to make the tests pass! And then put a reverse in the render.